masterlist.yaml

```diff
@@ -188,6 +188,11 @@ common:
         text: 'Delete or deactivate %1%.'
       - lang: ru
         text: 'Не забудьте отключить или удалить %1%.'
+  - &redundantPluginX
+    type: warn
+    content:
+      - lang: en
+        text: 'This plugin is redundant when using %1% and should be deactivated or deleted.'
```

Comment:
I dislike the imperative delete in `&alreadyInX`. Disabling is just as good and it's the safer option for the average user. However for e.g. `&AlreadyInUSKP` you must delete the mod and not just disable the plugin or else there might be weird conflicts with outdated loose files overwriting the USKP.
`&useOnlyOneX` fits well for some of these entries and is more precise. I'll change that.